### ISSUE TITLE
[10.0] Target loc back to main after we're done with 9.0

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/runtime#95024]
We should do this right after we snap rc1.

Basically do what we did in 8.0: https://github.com/dotnet/runtime/pull/90812

And then revert it after we ship 9.0 like we did here:
https://github.com/dotnet/runtime/pull/95014

**Comment by akoeplinger** [comment]
I don't think we need to follow https://github.com/dotnet/runtime/pull/90812 exactly since we (hopefully) don't need to disable OneLoc on main.

The only change needed should be setting `MirrorBranch: release/9.0` in the release/9.0 branch.

**Comment by ericstj** [comment]
Ping @carlossanlop - I think you did this?

**Comment by carlossanlop** [comment]
Yep, this is done. We're getting the Loc PRs created to the release/9.0 branch: https://github.com/dotnet/runtime/pull/106995

Let's keep this open so that we remember to restore the Loc branch to main after we release.

**Comment by ericstj** [comment]
Ok, since this is tracking work post-release I'll move to the 10.0.0 milestone.
